.travis.yml

```diff
@@ -0,0 +1,181 @@
+sudo: required
```

Comment:
Please set commit message in the following format:
issue: <#RM> - <....textual title....>
<blank line>
<Optional: more text describing the commit>

If RM issue does not already exist - please open a new one